templates/landingpage/index.html

```diff
@@ -49,19 +51,29 @@ <h1>Student/Parent</h1>
         </div>
 
         <div class="carousel-item">
-{#            TODO - add a background image url#}
             <img class="third-slide"
                  src="{% static 'landingpage/images/bg_4.jpg' %}"
                  alt="Third slide">
-{#            <div class="blur-bg"></div>#}
+			<div class="blur-bg"></div>
             <div class="container">
-                <div class="carousel-caption d-none d-md-block text-left">
+                <div class="carousel-caption justify-content-center d-flex h-50 d-md-block text-left">
                     <h1>Admission Staff</h1>
                     <p>Looking to manage millions of applications? Start now.</p>
-{#                    <p>#}
+
+                    <a href="{% url 'logIn:login_user' constant_ut_adminStaff %}"><button class="btn btn-lg btn-primary">Begin Here</button></a>
+                </div>
+            </div>
+        </div>
+         <div class="carousel-item">
+            <img class="third-slide"
+                 src="{% static 'landingpage/images/bg_4.jpg' %}"
+                 alt="Third slide">
+{#            <div class="blur-bg"></div>#}
+            <div class="container">
+                <div class="carousel-caption justify-content-center d-flex h-50 d-md-block text-left">
+                    <h1>Teacher and Mentor</h1>
+                    <p>Submit your letter now</p>
```

Comment:
Not urgent but instead of "Submit your letter now", how about "Were you asked to submit a recommendation letter? Check your email for details."

Reply:
solved